packages/test-runner-commands/test/send-mouse/sendMousePlugin.test.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import { platform } from 'os';
+import selenium from 'selenium-standalone';
+import { runTests } from '@web/test-runner-core/test-helpers';
+import { chromeLauncher } from '@web/test-runner-chrome';
+import { webdriverLauncher } from '@web/test-runner-webdriver';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { sendMousePlugin } from '../../src/sendMousePlugin';
+import { startSeleniumServer } from '../selenium-server';
+
+let seleniumServer: selenium.ChildProcess;
+
+describe('sendMousePlugin', function test() {
+  this.timeout(20000);
```

Comment:
Let's try the same timeout as `webdriver` launcher (and `selenium` launcher) tests.

```suggestion
  this.timeout(50000);
```

Reply:
Done.